release/air_tests/air_benchmarks/mlperf-train/resnet50_ray_air.py

```diff
@@ -543,7 +555,18 @@ def append_to_test_output_json(path, metrics):
         )
     )
 
-    write_metrics(train_loop_config["data_loader"], args, result, args.output_file)
+    try:
+        write_metrics(train_loop_config["data_loader"], args, result, args.output_file)
+    except OSError:
+        if not disk_error_expected:
+            raise
 
     if exc is not None:
-        raise exc
+        print(f"Raised exception: {exc}")
+        if not disk_error_expected:
+            raise exc
+        else:
+            # There is no way to get the error cause from the TuneError
+            # returned by AIR, so it's possible that it raised an error other
+            # than OutOfDiskError here.
+            pass
```

Comment:
Can we print the exception to reduce the chance that we have to debug a failed job without a failure message?